### DISCUSSION TITLE
Geany can't render Chinese characters?

### DISCUSSION THREAD

**Discussion by ghost** [geany/geany#3980] [General]
Windows included needed fonts for Chinese characters. Other text editors has no problem with rendering Chinese characters. The editor font is Consolas. I also have Consolas as the default font on Windows Notepad. But Windows Notepad has no problem rendering Chinese characters. It could be that Consolas doesn't support Chinese characters. I don't know. But font substitution should make it work out of the box. I don't know why it doesn't work on Geany.

**Comment by eht16** [comment]
Did you try another font?

To me it sounds like the issue in https://github.com/geany/geany/issues/3862. This issue also has some details about why fonts in Geany on Windows may look differentlyy than in other applications.

Can you confirm it is the same issue?

**Reply by ghost** [reply]
Font substitution is broken. Changing the font to Microsoft YaHei and it can display the Chinese characters just fine.
